### PR TITLE
Website: New layout

### PR DESCRIPTION
**Pull request by paselem**
- Update Layout
- Add EULA and Privacy Statement
- New screenshots

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -183,42 +184,13 @@ <h2>Plotting support</h2>
...
-                <img src="assets/images/pool-home.png" class="feature-image">
-            </div>
-            <div class="col-sm-7 col-sm-push-1">
-                <h2 style="font-weight:lighter; color:#333333">Heatmap visualizations</h2>
-                <p style="color:gray">This is some cool feature that lets you do some stuff</p>
-            </div>
+    <footer class="footer">
+        <div class="container">
+            <span class="text-muted">TODO: Add attributions to Font Awesome and other stuff here...</span>
+            <p>&copy; 2017-2018 Company, Inc. &middot; <a href="#">Privacy</a> &middot; <a href="#">Terms</a></p>
```
Comment (ascobie) [inline index.html]: do you really want the TODO:  here?

File: index.html [hunk 2]
```diff
@@ -8,82 +8,170 @@
...
+                        <img src="assets/images/job-running-tasks-graph.png" class="screenshot">
+                    </a>
+                </div>
+            </div>
         </div>
     </div>
+
+    <footer class="footer">
+        <div class="container">
+            <span class="text-muted">TODO: Add attributions to Font Awesome and other stuff here...</span>
```
Comment (timotheeguerin) [inline index.html]: Todo?

File: index.html [hunk 3]
```diff
@@ -8,82 +8,170 @@
...
+            <p>&copy; 2017-2018 Company, Inc. &middot; <a href="#">Privacy</a> &middot; <a href="#">Terms</a></p>
```
Comment (timotheeguerin) [inline index.html]: Link?

File: index.html (hunk 2 above)
Comment (ascobie) [reply index.html]: snap .... :)

**Pull request merged**